src/safeds/data/image/containers/_image.py

```diff
@@ -306,6 +308,38 @@ def adjust_brightness(self, factor: float) -> Image:
         image_copy._image = ImageEnhance.Brightness(image_copy._image).enhance(factor)
         return image_copy
 
+    def add_gaussian_noise(self, standard_deviation: float) -> Image:
+        """
+        Add Gaussian noise to the image.
+
+        Parameters
+        ----------
+        standard_deviation : float
+            The standard deviation of the Gaussian distribution.
+
+        Returns
+        -------
+        result : Image
+            The image with added Gaussian noise.
+        """
+        if standard_deviation < 0:
+            raise ValueError("Standard deviation has to be 0 or bigger.")
```

Comment:
I think `OutOfBoundsError(standard_deviation , name="standard_deviation", lower_bound=ClosedBound(0))` would be more fitting